auth-web/src/components/datatable/BaseVDataTable.vue

```diff
@@ -201,17 +222,21 @@ export default defineComponent({
     }
 
     watch(() => state.sortedItems, () => {
-      if (props.setItems && props.pageHide) {
+      if (props.setItems && !props.observerCallback) {
         state.visibleItems = state.sortedItems.slice(0, perPage.value)
         firstItem.value = state.visibleItems[0]
         currentPage.value = 1
         reachedEnd.value = false
         scrollToTop()
       }
+      if (props.observerCallback) {
```

Comment:
Will combine this with 225 in the future, probably return function, so it doesn't need to check for `props.observerCallback`